analyser/waveform.py

```
@@ -37,17 +34,17 @@
     sys.exit("Waveform storage bucket does not exist!")
 
 
-def get_next_waveform_previews(id_token, batch_size=20):
+def get_next_waveform_previews(batch_size=20):
     res = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)